components/application/validators.js

```diff
@@ -40,3 +40,7 @@ export const adminApplicationPatchStatusValidator = [
     }),
     body("message").optional().trim().escape()
 ]
+
+export const paymentSizeValidator = [
+    query("requestedLoanAmount").exists().isInt()

```

Comment:
This is fine, but if there's ever the option of "rewing loans" (paying an old loan with a new loan) it should be float